src/vorta/views/exclude_dialog.py

```diff
@@ -0,0 +1,410 @@
+            QMessageBox.critical(
+                self.parent(),
+                'Error',
+                'This exclusion already exists.',
+            )
+            self.removeRow(index.row())
+            return False
+
+        return super().setData(index, value, role)
+
+
+class ExcludeDialog(ExcludeDialogBase, ExcludeDialogUi):
+    def __init__(self, profile, parent=None):
+        super().__init__(parent)
+        self.setupUi(self)
+        self.profile = profile
+        self.allPresets = {}
+        self.setWindowModality(Qt.WindowModality.ApplicationModal)
+
+        self.buttonBox.rejected.connect(self.close)
+
+        self.customExclusionsModel = MandatoryInputItemModel()
+        self.customExclusionsList.setModel(self.customExclusionsModel)
+        self.customExclusionsModel.itemChanged.connect(self.custom_item_changed)
+        self.customExclusionsList.setEditTriggers(QAbstractItemView.EditTrigger.NoEditTriggers)
+        self.customExclusionsList.setSelectionMode(QAbstractItemView.SelectionMode.ExtendedSelection)
+        self.customExclusionsList.setFocusPolicy(Qt.FocusPolicy.NoFocus)
+        self.customExclusionsList.setAlternatingRowColors(True)
+        self.customExclusionsList.setStyleSheet(
+            '''
+            QListView::item {
+                padding: 10px 0px;
+                border-bottom: .5px solid black;
+            }
+            QListView::item:selected {
+                background-color: palette(highlight);
+            }
+
+        '''
+        )
+        self.customExclusionsListDelegate = QStyledItemDelegate()
+        self.customExclusionsList.setItemDelegate(self.customExclusionsListDelegate)
+        self.customExclusionsListDelegate.closeEditor.connect(self.custom_pattern_editing_finished)
+        # allow removing items with the delete key with event filter
+        self.installEventFilter(self)
+        # context menu
+        self.customExclusionsList.setContextMenuPolicy(Qt.ContextMenuPolicy.CustomContextMenu)
+        self.customExclusionsList.customContextMenuRequested.connect(self.custom_exclusions_context_menu)
+
+        self.exclusionPresetsModel = QStandardItemModel()
+        self.exclusionPresetsList.setModel(self.exclusionPresetsModel)
+        self.exclusionPresetsModel.itemChanged.connect(self.preset_item_changed)
+        self.exclusionPresetsList.setEditTriggers(QAbstractItemView.EditTrigger.NoEditTriggers)
+        self.exclusionPresetsList.setFocusPolicy(Qt.FocusPolicy.NoFocus)
+        self.exclusionPresetsList.setAlternatingRowColors(True)
+        self.exclusionPresetsList.setStyleSheet(
+            '''
+            QListView::item {
+                padding: 10px 0px;
+                border-bottom: .5px solid black;
+            }
+            QListView::item:selected {
+                background-color: palette(highlight);
+            }
+            QListView::item::icon {
+                padding-right: 10px;
+            }
+        '''
```

Comment:
Why do we even need to customly style the widgets?
And when you use the same style for multiple lists you can store the style sheet in a variable and reuse it.

Reply:
I didn't find a way to style these elements other than this one.

Yes, I can use the same string again.

Reply:
> I didn't find a way to style these elements other than this one (border, icon, item).

Yeah, but why to you want to style them? You override the Qt style used on the different platforms. On KDE Qt apps will use the Breeze theme which results in very nice looking list items, on Gnome one might use Kvantum.

Reply:
![image](https://github.com/borgbase/vorta/assets/41837037/b7320a3c-c249-4f47-9dbe-3fdef4370693)
It looked too packed for me, that's why I added the styles. Does it look good for you by default? Maybe then I can remove them.

Reply:
> It looked too packed for me, that's why I added the styles.

That's true but that's how the cerators of the oxygen style have chosen to design it. Outside my dev environment Vorta's lists look similar to this:

![grafik](https://github.com/borgbase/vorta/assets/62463991/9de5ad0f-8ce9-42d1-898f-97e386527b1c)